setup/Makefile

```
@@ -13,7 +13,10 @@ build-hermes:
 build-relayer:
 		cd $(APP_DIR)/neutron-query-relayer/ && make build-docker
 
-build-all: build-gaia build-neutron build-hermes build-relayer
+build-slinky:
+		@docker buildx build --build-context app=$(APP_DIR)/slinky -t skip-mev/slinky-e2e-oracle -f dockerbuilds/Dockerfile.slinky .
```

```suggestion
		@docker buildx build --load --build-context app=$(APP_DIR)/slinky -t skip-mev/slinky-e2e-oracle -f dockerbuilds/Dockerfile.slinky .
```

to avoid warning after the build process
```
WARNING: No output specified with docker-container driver. Build result will only remain in the build cache. To push result image into registry use --push or to load image into docker use --load
```